src/commands/registries/deleteRemoteImage.ts

```diff
@@ -13,7 +13,11 @@ import { registryExperience } from '../../utils/registryExperience';
 
 export async function deleteRemoteImage(context: IActionContext, node?: UnifiedRegistryItem<CommonTag>): Promise<void> {
     if (!node) {
-        node = await registryExperience(context, ext.registriesTree, { include: 'commontag', exclude: ['githubRegistryTag', 'dockerHubTag'] }, false);
+        node = await registryExperience(context, ext.registriesTree,
+            {
```

Comment:
```suggestion
        node = await registryExperience(context, ext.registriesTree, {
```

Reply:
This will also adjust the indentation of the next three lines, should be automatic when you save